packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -5695,7 +6225,8 @@ function buildProviderConfig(config: Partial<ProviderConfig> = {}) {
     chainId: '1337',
     id: undefined,
     nickname: undefined,
-    rpcTarget: undefined,
+    rpcTarget:
+      config.type === NetworkType.rpc ? 'http://doesntmatter.com' : undefined,
```

Comment:
This was updated to provide a valid provider config when just `NetworkType.rpc` was passed in. This was convenient for keeping the custom RPC tests brief.